pkg/network/http/model.go

```diff
@@ -50,7 +51,7 @@ func (tx *httpTX) Path() string {
 
 	i++
 
-	for j = i; j < len(b) && b[j] != ' '; j++ {
+	for j = i; j < len(b) && b[j] != ' ' && b[j] != '?'; j++ {
```

Comment:
Nit: I think it's worth updating the comment above to indicate that this function removes query variables from the URL before returning it

Reply:
Good point, I'll update it 👍